FactoryWeather/App/VIP Settings/SettingsRouter.swift

```diff
@@ -9,13 +9,13 @@
 import Foundation
 
 protocol SettingsRoutingLogic {
-    func getNewWeather(selectedLocation: Location?)
-    func dismissSettingsScene()
+    func requestNewWeatherData(selectedLocation: Location?)
+    func unwindBack()
 }
 
 protocol SettingsRouterDelegate: class {
-    func getNewWeather(selectedLocation: Location?)
-    func dismissSettingsScene()
+    func requestNewWeatherData(selectedLocation: Location?)
```

Comment:
To make it more understandable for consumers, we should use the following format when naming methods in the `RouterDelegate` protocols: 

`XRouterY` where `X` is the scene identifier and `Y` is the action descriptor.

In example:
`requestNewWeatherData` would become `settingsRouterRequestedNewWeatherData`.

Reply:
Yeah.